source/blog/2018-05-11-the-emberjs-times-issue-46.md

```diff
@@ -57,15 +43,32 @@ App.AboutRoute = Ember.Route.extend({
     {{/each}}
   </ul>
 </script>
+```
 
-Using Ember-CLI instead has been recommended for years, but anyone still using this api should
+Using Ember-CLI instead has been recommended for years, but anyone still using this API should
 read the RFC and comment on why they might still need it.
 
+---
+
+## [EMBER CLI THINGS](#your-url-here)
+
 
 ---
 
-## [SOMETHING ELSE HERE](#your-url-here)
+## [Get involved with the new Ember guides](https://emberguides.stonecircle.io/release/)
+The Learning Team has been working on a new and improved version of the Guides app. It's now made with Ember.js and is more shiny than ever. The Guides app is almost ready for a proper release, but before we do so, we would love for you to [try it out](https://emberguides.stonecircle.io/release/). If you find any issues then please [report them](https://github.com/ember-learn/guides-app/issues).
+
+Or if you feel in a particulary good mood, look through existing issues in the [Guides app](https://github.com/ember-learn/guides-app/issues) or the [Guides source](https://github.com/ember-learn/guides-source/issues) and start contributing to the new guides.
+
+---
+
+## [Moar #EmberJS2018 blog posts wanted!](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html)
+
+We've already seen a lot of thoughtful blog posts from the Ember team's [call for community blog posts](https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html). But we're hoping for more! Have something to say?
+
+Write a blog post by **May 30th** to propose goals and direction for Ember in the remainder of 2018. The content of these posts will help the core team to draft their first Roadmap RFC.
 
+Looking for inspiration? Check out the [**#EmberJS2018** hashtag](https://twitter.com/search?q=%23EmberJS2018) on Twitter.
```

Comment:
We could also mention this repo that is collecting all current blog posts at the time: https://github.com/zinyando/emberjs2018-posts What do you think?

Reply:
👍 nice one, added it. this section could be moved up if you think it's higher priority